src/youtube/js/youtube.js

```diff
@@ -32,7 +32,7 @@ require("../../../html5-common/js/utils/constants.js");
     this.features = [ OO.VIDEO.FEATURE.VIDEO_OBJECT_SHARING_GIVE,
                       OO.VIDEO.FEATURE.BITRATE_CONTROL ];
     this.technology = OO.VIDEO.TECHNOLOGY.HTML5;
-    this.encodings = [OO.VIDEO.ENCODING.MP4];
+    this.encodings = [OO.VIDEO.ENCODING.YOUTUBE];
```

Comment:
Oh if you are asking if we need to include both then no. 
If we include both, it could create issues with other MP4 plugins.

There are corresponding changes on mjolnir that need review too.
Look for Pull request 154.